apps/api/src/app/shared/framework/paginated-ok-response.decorator.ts

```diff
@@ -0,0 +1,24 @@
+import { ApiExtraModels, ApiOkResponse, getSchemaPath } from '@nestjs/swagger';
```

Comment:
taken from this blog
https://aalonso.dev/blog/how-to-generate-generics-dtos-with-nestjsswagger-422g